task/prefetch-dependencies-oci-ta/0.1/prefetch-dependencies-oci-ta.yaml

```diff
@@ -325,6 +183,50 @@ spec:
       script: |
         #!/bin/bash
 
```

Comment:
Can we use this opt `set -euo pipefail` to make sure that we fail early, and then use `trap` to unregister? It's confusing to users to see script continues when there was a failures. It generates misleading output.

Reply:
I thought it was proposed in a different PR, but sure, I can enable it in a separate commit and as I mentioned in a different response to the PR *add the trap*.

Reply:
it was, but this PR changed the task completelly